DevIncubator_Autopark/AllCollections/MyStack.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace DevIncubator_Autopark.AllCollections
+{
+    public class MyStack<T> : IEnumerable<T>
+    {
+        public int Count => GetCount();
+
+        private T[] _stack;
+        private int _headItemIndex;
+
```

Comment:
The built in .NET Stack<T> has a constructor Stack<T>(Int32). A think, you can add that too.
`public MyStack(int size)`

Reply:
fixed